index.html

```diff
@@ -1,14 +1,13 @@
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
 </head>
+
 <body>
-    <h1>Page title</h1>
-    <ul>
-        <li>red</li>
-        <li>green</li>
-    </ul>
+    <h1>Hello world!</h1>
 </body>
+
 </html>
```

Comment:
Add [empty line in the end of file](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file)

Reply:
Got it, it's done.